index.html

```diff
@@ -41,8 +41,170 @@ <h1>Hey there. I’m Mariane <span class="h-span">I’m a software developer</sp
             </button>
         </div>
     </section>
-    <div class="myrecentwork">
-        <h1>My Recent Works</h1>
-    </div>
+    <section>
+        <h1 class="myrecentwork">My Recent Works</h1>
+        <img class="ind" src="images/Indicator.png" alt="">
+        <div class="work-container">
+            <div class="img-placer">
+                <img src="images/im1.png" alt="">
+            </div>
+            <div class="txt-info">
+                <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+                <ul class="languages">
+                    <li><a href="">Ruby on rails</a></li>
+                    <li><a href="">css</a></li>
+                    <li><a href="">JavaScript</a></li>
+                    <li><a href="">html</a></li>
+                </ul>
+                <button class="see-project" type="button">
+                    <span class="project-title">See Project</span> </button>
+            </div>
+        </div>
+        <div class="work-container">
+            <div class="img-placer">
+                <img src="images/im1.png" alt="">
+            </div>
+            <div class="txt-info">
+                <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+                <ul class="languages">
+                    <li><a href="">Ruby on rails</a></li>
+                    <li><a href="">css</a></li>
+                    <li><a href="">JavaScript</a></li>
+                    <li><a href="">html</a></li>
+                </ul>
+                <button class="see-project" type="button">
+                    <span class="project-title">See Project</span> </button>
+            </div>
+        </div>
```

Comment:
Great job @Tekhlay 🤝

- [ ] As stated in the project requirements, kindly align the elements of these individual cards with `display: flex`.